### PR TITLE
feat(faq-category-list): add 404 page

### LINKED ISSUE #293: When there is a network issue, 'No Bookings Found' is displayed, instead of an error
When booking api is down, there should be an error displayed, but this is shown instead:
2018-05-31-151909_2205x950_scrot

This might be confusing to users and might be generating extra customer service traffic. We should display something like "Network error. Please try again in a few minutes. For urgent inquiries don't hesitate to call customer service at {phone number}"

### PR DESCRIPTION
**Pull request by kantord**
fixes #293<br/><br/><br/><url>LiveURL: https://smartfaq-add-error-page-for-missing-faq.surge.sh</url>

### CODE REVIEW

File: src/StaticFAQ/FAQCategoryList.js [hunk 1]
```diff
@@ -182,6 +182,10 @@ class RawFAQCategoryList extends React.Component<Props> {
       return <StaticFAQError />;
     }
 
+    const FAQCategory = idx(rendererProps.props, _ => _.FAQCategory);
+
+    if (FAQCategory === null) return <div>Category not found</div>;
```
Comment (jalbertsr) [inline src/StaticFAQ/FAQCategoryList.js]: I think rendering the error for FAQs would be better: `src/StaticFAQ/StaticFAQError.js`

File: src/StaticFAQ/FAQCategoryList.js [hunk 2]
```diff
@@ -182,6 +182,12 @@ class RawFAQCategoryList extends React.Component<Props> {
       return <StaticFAQError />;
     }
 
+    const FAQCategory = idx(rendererProps.props, _ => _.FAQCategory);
```
Comment (jaroslav-kubicek) [inline src/StaticFAQ/FAQCategoryList.js]: you need to do it inside `rendererProps.props` condition, this would break Loader below

Comment (kantord) [reply src/StaticFAQ/FAQCategoryList.js]: fixed

**Comment by jalbertsr** [comment]
this issue fixes https://github.com/kiwicom/smart-faq/issues/590 not #293.

**Pull request merged**